src/z3c/saconfig/tests.py

```diff
@@ -7,36 +7,37 @@
 # Since the test exercise what happens with two different DSNs
 # locally, you need to also set up a different TEST_DSN2.
 
-# To test in twophase commit mode export TEST_TWOPHASE=True 
+# To test in twophase commit mode export TEST_TWOPHASE=True
 #
 # NOTE: The sqlite that ships with Mac OS X 10.4 is buggy.
 # Install a newer version (3.5.6) and rebuild pysqlite2 against it.
-
-
-import unittest
-import doctest
-import os
-
+from zope import component
+from zope.interface import registry
 from zope.testing import cleanup
 from zope.testing.cleanup import addCleanUp
 
-from zope import component
-from zope.component import registry
+import doctest
+import os
+import six
+import unittest
 import zope.component.eventtesting
```

Comment:
This seems wrong: PEP-8 suggests putting standard library imports (doctest, os, unittest) in a group above 3rd-party library imports (zope.*).